Changefeed backup/restore documentation (tool dump) (#12656)
ydb-platform/ydb: ydb/docs/en/core/reference/ydb-cli/export-import/_includes/file_structure.md

- The `scheme.pb` file describing the table structure and parameters in the [text protobuf](https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.text_format) format
- One or more `data_XX.csv` files with the table data in `csv` format, where `XX` is the file's sequence number. The export starts with the `data_00.csv` file, with a next file created whenever the current file exceeds 100 MB.

> - Directories for describing [changefeeds](https://ydb.tech/docs/en/concepts/cdc). The directory name corresponds to the name of the changefeed. Inside the directory, you can find:

Reviewer: ```suggestion
- Directories describing the [changefeeds](https://ydb.tech/docs/en/concepts/cdc). Directory names match the names of the changefeeds. Each directory contains the following files:
```

>   - The file changefeed_description.pb, which contains information about the changefeed in [text protobuf](https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.text_format) format.

Reviewer: ```suggestion
  - The `changefeed_description.pb` file describing the changefeed in the [text protobuf](https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.text_format) format.
```

>   - The file topic_description.pb, which contains information about the underlying topic in [text protobuf](https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.text_format) format.

Reviewer: ```suggestion
  - The `topic_description.pb` file describing the underlying topic in the [text protobuf](https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.text_format) format.
```
